packages/frontend-v2/components/Plan/ReqErrorModal.tsx

```diff
@@ -0,0 +1,165 @@
+import { WarningIcon } from "@chakra-ui/icons";
+import {
+  Button,
+  Flex,
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+  useDisclosure,
+} from "@chakra-ui/react";
+import {
+  assertUnreachable,
+  courseToString,
+  INEUPrereqError,
+  ScheduleCourse2,
+} from "@graduate/common";
+import { useState } from "react";
+
+interface ReqErrorModalProps {
+  course: ScheduleCourse2<string>;
+  preReqErr: INEUPrereqError | undefined;
+  coReqErr: INEUPrereqError | undefined;
+}
+
+export const ReqErrorModal: React.FC<ReqErrorModalProps> = ({
+  course,
+  coReqErr,
+  preReqErr,
+}) => {
+  const [page, setPage] = useState(0);
+
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  return (
+    <Flex
+      justifySelf="stretch"
+      alignSelf="stretch"
+      _hover={{
+        background: "primary.red.main",
+        fill: "white",
+        svg: { color: "white" },
+      }}
+      _active={{ background: "primary.red.900" }}
+      onClick={() => {
+        onOpen();
+      }}
+    >
+      <WarningIcon
+        color="primary.red.main"
+        width="2rem"
+        alignSelf="center"
+        alignItems="center"
+        justifySelf="center"
+        transition="background 0.15s ease"
+      />
+      <Modal isOpen={isOpen} onClose={onClose}>
+        <ModalOverlay />
+        <ModalContent>
+          <ModalHeader
+            textAlign="center"
+            backgroundColor="#D9D9D9"
+            borderTopRadius="md"
+          >
+            Course Errors for {courseToString(course)}
+          </ModalHeader>
+          <ModalBody>
+            <Text fontSize="sm" mb="sm" color="red" textAlign="center">
+              It looks like you are missing prerequisites for this course
+            </Text>
+            {coReqErr && page == 0 && (
+              <Flex direction="column">
+                <Text fontWeight="semibold" mb="xs" textAlign='center'>
+                  CoReq Errors
+                </Text>
+                <ParseCourse course={coReqErr} />
+              </Flex>
+            )}
+            {preReqErr && (page == 1 || !coReqErr) && (
+              <Flex direction="column">
+                <Text fontWeight="semibold" mb="xs" textAlign='center'>
+                  PreReq Errors
+                </Text>
+                <ParseCourse course={preReqErr} />
+              </Flex>
+            )}
+          </ModalBody>
+
+          <ModalFooter>
+            <Flex
+              justifyContent="space-between"
+              width="100%"
+              alignItems="center"
+            >
+              <Button
+                variant="solidRed"
```

Comment:
I think from a UI perspective, we should probably keep red buttons for dangerous actions (closing this modal doesn't have any destructive effects).
```suggestion
                variant="solidBlue"
```